msmq/0.3.9/modules/ROOT/nav.adoc

```diff
@@ -1,5 +1,4 @@
 .xref:index.adoc[MSMQ Connector]
-* xref:index.adoc[About MSMQ Connector]
+* xref:index.adoc[]
```

Comment:
```suggestion
* xref:index.adoc[About MSMQ Connector]
```

Reply:
need to hardcode the nav so all connector navs are consistent because the H1s are usually different
https://github.com/mulesoft/docs-connectors/blob/latest/connectors-home/modules/ROOT/pages/_connector-template-files/connector-template-files-nav.adoc